src/perl/bin/sample_udf_updater

```diff
@@ -0,0 +1,240 @@
+#! /usr/bin/env perl
+
+package sample_udf_updater;
+
+use strict;
+use warnings;
```

Comment:
You don't really need to do `use strict` and `use warnings` if you `use Moose` as Moose always does that bit for you